# EKF State Message Structure

# Position
float64 x
float64 y
float64 z

# Attitude
float64 roll
float64 pitch
float64 yaw

# Velocity
float64 vx
float64 vy
float64 vz

# Acceleration Bias
float64 bias_ax
float64 bias_ay
float64 bias_az

# Gyro Bias
float64 bias_gx
float64 bias_gy
float64 bias_gz
